Expensify/App
# Only ask for last 4 digits of SSN

### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/166493

### Tests
- Make sure you have a bankAccount with addressName = Expensify Wallet and addressState = false (a silver wallet)
- Go to `/enable-payments` in e.cash
- Select any form, it will give you a link
- Open that link in your phone and do the checks
- Go back to the browser, you should be in this page:
![image](https://user-images.githubusercontent.com/521248/123337889-20fdf880-d505-11eb-917d-956ff26c1c42.png)
- Complete everything
- Check it works (disclaimer: I could not make the idology tests pass in dev, but they should work in stg/prod).

### Tested On


### Screenshots
![image](https://user-images.githubusercontent.com/521248/123338043-6a4e4800-d505-11eb-97a8-c33e61125d63.png)
